uftrace.c

```diff
@@ -1130,6 +1188,49 @@ static void update_subcmd(struct uftrace_opts *opts, char *cmd)
 		opts->mode = UFTRACE_MODE_INVALID;
 }
 
+static int print_subhelp(int beforekey)
+{
+	int retval = -1;
+	switch (beforekey) {
+	case UFTRACE_MODE_RECORD:
+		pr_out(uftrace_record_usage);
+		break;
+	case UFTRACE_MODE_REPLAY:
+		pr_out(uftrace_replay_usage);
+		break;
+	case UFTRACE_MODE_REPORT:
+		pr_out(uftrace_report_usage);
+		break;
+	case UFTRACE_MODE_LIVE:
+		pr_out(uftrace_live_usage);
+		break;
+	case UFTRACE_MODE_GRAPH:
+		pr_out(uftrace_graph_usage);
+		break;
+	case UFTRACE_MODE_INFO:
+		pr_out(uftrace_info_usage);
+		break;
+	case UFTRACE_MODE_DUMP:
+		pr_out(uftrace_dump_usage);
+		break;
+	case UFTRACE_MODE_RECV:
+		pr_out(uftrace_recv_usage);
+		break;
+	case UFTRACE_MODE_SCRIPT:
+		pr_out(uftrace_script_usage);
+		break;
+	case UFTRACE_MODE_TUI:
+		pr_out(uftrace_tui_usage);
+		break;
+	case UFTRACE_MODE_INVALID:
+	default:
+		retval = -3;
+		break;
+	}
+
+	return (retval);
```

Comment:
We don't use parenthesis for return statement.

Reply:
@RndelQndel `return retval`